### ISSUE TITLE
How to use this approach in case of resnet architecture

### ISSUE THREAD

**Issue by surekhag28** [kumar-shridhar/PyTorch-BayesianCNN#51]
This is not an issue but not sure where should I ask this question hence thought of posting it here.
From the models I can see that it is very linear and do not need forward function as it has common wrapper. I tried to follow this approach for resnet architecture but I am kind of end up getting always 14% accuracy.
Will it be possible for you to provide sample example for the same (just the model)?

**Comment by lvyilin** [comment]
I got the same problem while implement in resnet50 and train by imagenet. It got about 23% acc in 40 epochs. It seems that hard to train in deeper architecture.
